### ISSUE TITLE
Some suggestion on this amazing plugin

### ISSUE THREAD

**Issue by vcvrackideas** [baconpaul/airwin2rack#16]
Hi and thank you for doing such big work! I've just played with this and got some random thoughts that could be useful to implement from an user experience perspective. 
1. In and out gains. Some levels (oscillators, filters, whatever) in VCV are really hot and it's not logical to add another vca before and maybe after that to manipulate levels to avoid clipping or distortion.
2. Polyphonic input handling. I know it may hard to implement polyphony in originally vst algorithms but Grayscale Supercell has a nice solution on this point - it actually do poly to mono conversion at the input of plugin.
3. Favorites effects (some thing that old VCV browser had in the same menu as you implemented here). So it actually can sorting out effects not by groups but by user liking (guess it's right word here). For example I love MatrixVerb but not take a hand on Nonlinearverb. So I could put Matrix verb into favorites group and just switching between favorites without loosing a flow or something like that.
4. Panic button. Just a button that would stop incoming signal and just restart DSP. That idea came to my mind when those aggressive (but overally good) reverbs did come to endless feedback and distort then. Sometimes it just needs to change incoming signal or do some tweaks to get effect sound right. So this button would help to "restart" all of that chaotic mess without switching cables.

Cheers!

**Comment by baconpaul** [comment]
Thanks!

1 and 4 are good ideas but I need to figure out where on the panel they would go. 3 is a good idea but not easy to code. I’ll think about them for sure!

but 2 is there now! By default the fx is mono and does a poly sum to mono at Input but in the menu is a poly option which runs an independent version of the fx per channel.

**Comment by vcvrackideas** [comment]
Thanks! Also stacking that I had proposed here https://github.com/surge-synthesizer/surge-rack/issues/829 could also be applied to Airwindows to due similar UI as in SurgeXT if you like the idea of course

**Comment by baconpaul** [comment]
Yeah that evoked into auto hookup and I’ll add to both when I do it!

**Comment by baconpaul** [comment]
Heya! I knot's been sitting here open for a bit apologies. Been doing a bit of work on this module other than regular fx-sweeps this week and remembered this issue, so added a PanicReset menu item and also an input and output attenuator.

I really don't think I'm going to do a separate favorites thing but I'll rename the issue to indicate that's the only outstanding request.

Should e in a nightly today or tomorrow and in the library soon enough.

![Atten](https://github.com/baconpaul/airwin2rack/assets/13387561/dd73fbb7-5ed7-4aea-86e1-f95ddb7081de)

**Comment by erroreyes** [comment]
I came here to give support for #3, a way to create a list of favorites. 

I just loaded the CLAP plugin for the first time, noticed the the collections and immediately thought of being able to tag favorite plugins in some manner since the "Recommended" collection does not contain many of the Airwindows plugins that I consider essential to me.

I think this would be a feature that would be highly appreciated by many users, especially as a way to manage and make sense of a collection of over 200 plugins, and will aid from a UX prespective to lessen menu-diving.

**Comment by baconpaul** [comment]
I added this in consolidated daw plugin. So just a reminder that this issue is here for the rack side
